src/atomicdex/pages/qt.wallet.page.cpp

```diff
@@ -270,6 +287,19 @@ namespace atomic_dex
                 {
                     j_out["withdraw_answer"]["fee_details"]["amount"] = j_out["withdraw_answer"]["fee_details"]["miner_fee"];
                 }
+                
+                // Add fees amount in fiat currency.
+                auto fee = j_out["withdraw_answer"]["fee_details"]["amount"];
```

Comment:
Spurious automatic conversion to `std::string` will rather prefer to use `get<std::string>()` to be sure.